docs/lesiaturuta/PokemonProject/scripts/checkScroll.js

```diff
@@ -0,0 +1,12 @@
+const checkScroll = () => {
```

Comment:
I assume you intend to make a bottom-sticky footer? I could be done without js

Reply:
fixrd